lasagne/layers/helper.py

```diff
@@ -290,9 +290,10 @@ def get_output_shape(layer_or_layers, input_shapes=None):
         return all_shapes[layer_or_layers]
 
 
-def get_all_params(layer, **tags):
+def get_all_params(layer, unwrap_shared=True, **tags):
     """
-    Returns a list of Theano shared variables that parameterize the layer.
+    Returns a list of Theano shared variables (default) or expressions that
+    parameterize the layer.
```

Comment:
Again, I'd omit `(default)`. Also check how this renders in the overview table at http://lasagne.readthedocs.org/en/latest/modules/layers.html. Maybe it has to be kept on one line. Note that the initial sentence of `Layer.get_params()` has to be adapted as well.